tests/app/delivery/test_send_to_providers.py

```diff
@@ -978,8 +978,8 @@ def test_notification_with_bad_file_attachment_url(mocker, notify_db, notify_db_
     personalisation = {
         "file": document_download_response(
             {
-                "direct_file_url": "file://foo.bar/file.txt" if sending_method == "attach" else "http://foo.bar/file.txt",
-                "url": "file://foo.bar/file.txt" if sending_method == "link" else "http://foo.bar/file.txt",
+                "direct_file_url": "file://foo.bar/file.txt",
```

Comment:
`check_file_url` rejects any URL which does not start by `http`, thus for better clarity, I suggest following change:
```suggestion
                "direct_file_url": "http://foo.bar/file.txt",
```

Reply:
good point, thanks for the suggestion!